Agreements/Airport/BB_Trent.txt

```diff
@@ -1,2 +1 @@
-;FixMe LUVUM/ADMEX etc?
-COPX:EGBB:*:TNT:*:*:BBAPP:Trent:8000:*:^TNT
\ No newline at end of file
+COPX:EGBB:*:TNT:*:*:BBAPP:Trent:8000:*:^TNT
```

Comment:
This file can be deleted - anything going to TNT goes via LUVUM, and should go to the Stafa sector, not Trent.

Reply:
Deleted in d80d1d97b9fa2f4b8ae188d88bd61cc08ae43d99